devel/uboot-tools/Portfile

```diff
@@ -5,7 +5,7 @@ PortGroup           github 1.0
 
 name                uboot-tools
 github.setup        u-boot u-boot 2020.07 v
-revision            0
+revision            1
```

Comment:
The revision bump is not needed: the port does not need to rebuild if it was already able to build.